code/lightna/magento-os-frontend/template/page/header/account.phtml

```diff
@@ -0,0 +1,48 @@
+<?php
+/** @var Lightna\Magento\Data\Session $session */
+$isLoggedIn = $session->user->groupId !== 0;
+$iconClass = $isLoggedIn ? 'user__filled' : 'user__outline';
+?>
+<div class="icon-dropdown dropdown">
+    <div class="header__icon">
+        <a href="/customer/account" class="icon <?= $iconClass ?> cursor-default"></a>
```

Comment:
click on header icon does work when you logged in

Reply:
I did that on purpose, click on icon works only on mobile, on desktop there is only hover to show dropdown

Reply:
hm, confusing, I expected to go to my account

Reply:
I was thinking it is up to me 😁

Reply:
done: icon is clickable on desktop as well

Reply:
right, it's up to, nobody require you listening to me :laughing: 
I was thinking you just always agree with me))